tests/fsharpqa/Source/Conformance/DeclarationElements/LetBindings/Basic/E_AttributesOnLet01.fs

```diff
@@ -4,7 +4,10 @@
 // Regression test for FSharp1.0:3744 - Unable to apply attributes on individual patterns in a tupled pattern match let binding - Implementation doesn't match spec
 // Test against error emitted when attributes applied within pattern
 
-//<Expects id="FS0683" span="(11,6-11,33)" status="error">Attributes are not allowed within patterns</Expects>
+//<Expects id="FS0683" span="(14,6-14,27)" status="error">Attributes are not allowed within patterns</Expects>
+//<Expects id="FS0842" span="(14,8-14,23)" status="error">This attribute is not valid for use on this language element</Expects>
```

Comment:
This is better considering it is now showing multiple errors on both attributes along with better ranges.

The only minor knit I have is that it is also giving an error saying the "attribute is not valid for use on this language element" - but the real error is that attributes are just not allowed in patterns.